app/helpers/events_helper.rb

```diff
@@ -99,4 +99,13 @@ def past_category_name(type_id)
     t "event_types.#{type_id}.name.past",
       default: "Past " + pluralised_category_name(type_id)
   end
+
+  def display_no_ttt_events_message?(performed_search, events, event_search_type)
+    train_to_teach_id = GetIntoTeachingApiClient::Constants::EVENT_TYPES["Train to Teach event"]
+
+    searching_for_ttt_or_all_events = [train_to_teach_id.to_s, ""].include? event_search_type
```

Comment:
I don't feel that strongly about it, but I had to do a double take to figure out why you're checking for an empty string here (its clearer when you know that `event_search_type` comes from a query string param, or lack thereof). IMO it may be easier to read if you split them:

```
searching_for_ttt = train_to_teach_id == event_search_type
searching_for_all = event_search_type.blank?
```

Reply:
I've changed it as suggested :)